db/seeds/development/all.seeds.rb

```diff
@@ -1 +1,8 @@
 FactoryGirl.create(:user, :confirmed, email: "user@example.com")
+10.times do |id|
```

Comment:
and again pls use ruby style guides)) empty line before and after block can make you code more readable.